### PR TITLE
slide-switch: Update to 0.9.5

### PR DESCRIPTION
**Pull request by jefferyto**
Maintainer: me
Compile tested: armvirt-64, 2019-08-01 snapshot sdk
Run tested: manually tested on 18.06.2, gl-ar750

Description:
Signed-off-by: Jeffery To <jeffery.to@gmail.com>

### CODE REVIEW

File: utils/slide-switch/Makefile [hunk 1]
```diff
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=slide-switch
-PKG_VERSION:=0.9.4
+PKG_VERSION:=0.9.5
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/jefferyto/openwrt-slide-switch.git
```
Comment (neheb) [inline utils/slide-switch/Makefile]: .git at the end is unnecessary.

File: utils/slide-switch/Makefile [hunk 2]
```diff
@@ -8,15 +8,15 @@
...
-PKG_SOURCE_VERSION:=0.9.4
+PKG_MIRROR_HASH:=ac61aea3ce620364285de5525635999aa8b463c4070da6bce134278ff92a433c
+PKG_SOURCE_VERSION:=0.9.5
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
```
Comment (neheb) [inline utils/slide-switch/Makefile]: Remove

File: utils/slide-switch/Makefile [hunk 3]
```diff
@@ -8,15 +8,15 @@
...
 PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.xz
```
Comment (neheb) [inline utils/slide-switch/Makefile]: Remove.

File: utils/slide-switch/Makefile [hunk 4]
```diff
@@ -8,15 +8,15 @@
...
 PKG_NAME:=slide-switch
-PKG_VERSION:=0.9.4
+PKG_VERSION:=0.9.5
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/jefferyto/openwrt-slide-switch.git
-PKG_SOURCE_VERSION:=0.9.4
+PKG_MIRROR_HASH:=ac61aea3ce620364285de5525635999aa8b463c4070da6bce134278ff92a433c
+PKG_SOURCE_VERSION:=0.9.5
```
Comment (neheb) [inline utils/slide-switch/Makefile]: can be written as PKG_SOURCE_VERSION:=$(PKG_VERSION)

File: utils/slide-switch/Makefile (hunk 1 above)
Comment (jefferyto) [reply utils/slide-switch/Makefile]: This is the official URL GitHub gives to clone using https; I prefer using this URL.

The fact that the `.git` at the end is not required is an unofficial convenience that can be revoked at any time.

Comment (neheb) [reply utils/slide-switch/Makefile]: Fair enough.

**Pull request merged**